clients/python/llmengine/model.py

```diff
@@ -110,6 +111,13 @@ def create(
             gpus (`Optional[int]`):
                 Number of gpus each worker should get, e.g. 0, 1, etc. Can be inferred from the model size.
 
+            nodes_per_worker (`int`):
+                Number of nodes per worker. Used to request multinode serving. This must be greater than or equal to 1.
+                Controls how many nodes to dedicate to one instance of the model.
+                Any compute resource requests (i.e. cpus, memory, storage) apply to each node, thus the total resources
+                allocated are multiplied by this number. This is useful for models that require more memory than a single node can provide.
+                Note: autoscaling is not supported for multinode serving.
```

Comment:
we'd add it in the future? since LWS does support HPA

Reply:
is it because we don't have istio metrics? we do have vLLM metrics

Reply:
I didn't add HPA to keep things simple here; I want to be more certain that the LWS replicas can start up properly (I found issues with starting up LWS in the past with random NCCL errors that sometimes happened). Also, I don't think our current use case would require autoscaling. We can certainly add autoscaling to LWS if we need though. (also yeah we probably don't have istio metrics)